miner/worker.go

```diff
@@ -1166,7 +1171,7 @@ func (w *worker) commitWork(interrupt *int32, noempty bool, timestamp int64) {
 	}
 	// Fill pending transactions from the txpool
 	w.fillTransactions(interrupt, work)
-	w.commit(work.copy(), w.fullTaskHook, true, start)
```

Comment:
There are other 2 spots that `w.commit(work.copy(), ...)` will be executed.
Seems `work.copy()` are all unnecessary?

Reply:
I think yes, it copies twice